README.md

```diff
@@ -6,6 +6,8 @@
 [![Coverage Status](https://coveralls.io/repos/github/cabol/nebulex/badge.svg?branch=master)](https://coveralls.io/github/cabol/nebulex?branch=master)
 [![Inline docs](http://inch-ci.org/github/cabol/nebulex.svg)](http://inch-ci.org/github/cabol/nebulex)
 [![Hex Version](https://img.shields.io/hexpm/v/nebulex.svg)](https://hex.pm/packages/nebulex)
+[![Downloads](https://img.shields.io/hexpm/dt/nebulex.svg)](https://hex.pm/packages/nebulex)
```

Comment:
Can we remove this one, I think it is needed.

Reply:
> Can we remove this one, I think it is needed.

Done.